source/overview/product.rst

```diff
@@ -2,70 +2,154 @@
-- Threaded messaging, emoji and custom emoji
+- Threaded messaging, emojis and custom emojis
 - Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`_ and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`_
-- Extensive integration support via `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`_ and `third party extensions <https://about.mattermost.com/default-app-directory/>`_
+- Extensive integration support through `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`_ and `third party extensions <https://about.mattermost.com/default-app-directory/>`_
 - Easily scalable from dozens to hundreds of users
 - New improvements released every two months
-- Languages include U.S. English, Chinese (Simplified & Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Russian, Turkish, and Spanish
+- Languages include U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Russian, Turkish, and Spanish.
 
-To get started, `download the open source Mattermost Team Edition server <https://about.mattermost.com/download>`_ under an MIT license.
+Mattermost Enterprise Edition E10 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enterprise Edition E10  is a commercial *virtual campus* for your teams, 
+groups,
+and departments across multiple projects and scalable to thousands of users. 
+This is a commercial extension to the open source Team Edition with commercial support, 
+corporate directory integration through Active Directory/LDAP, 
+advanced access controls, 
+security, 
+and custom branding options. 
+
+.. note::
+Get started with a `free trial license <https://about.mattermost.com/trial/>`_. You can later purchase online with a `credit card <https://about.mattermost.com/pricing/>`_ or by `bank transfer <https://about.mattermost.com/quotation/>`_.
+
+Enterprise Edition E10 Highlights
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Mattermost Enterprise Edition E10** is a commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
+E10 offers security, 
+configuration, 
+and access control options for larger organizations, including:
 
 - Active Directory / LDAP single-sign-on
-- Encrypted push notifications via HPNS
+- Encrypted push notifications through HPNS
 - Multi-factor authentication
 - Tools for custom branding
 - Advanced access control policy
-- Next business day support via online ticketing system
-- Scale to handle thousands of users
+- Next business day support through online ticketing system
+- Scale to handle thousands of users.
```

Comment:
Should this be changed to "Ability to scale" or some other sentence that reads better with the intro

Reply:
no, at least not to my sense. this is an extension of the misused bullets